TESTOWNERS

```diff
@@ -220,9 +221,9 @@
 /dev/devicelab/bin/tasks/flutter_gallery_macos__start_up.dart @cbracken @flutter/desktop
 /dev/devicelab/bin/tasks/flutter_gallery_win_desktop__compile.dart @yaakovschectman @flutter/desktop
 /dev/devicelab/bin/tasks/flutter_gallery_win_desktop__start_up.dart @yaakovschectman @flutter/desktop
-/dev/devicelab/bin/tasks/flutter_tool_startup__linux.dart @jensjoha @flutter/tool
-/dev/devicelab/bin/tasks/flutter_tool_startup__macos.dart @jensjoha @flutter/tool
-/dev/devicelab/bin/tasks/flutter_tool_startup__windows.dart @jensjoha @flutter/tool
+/dev/devicelab/bin/tasks/flutter_tool_startup.dart @jensjoha @flutter/tool
+/dev/devicelab/bin/tasks/flutter_tool_startup.dart @jensjoha @flutter/tool
+/dev/devicelab/bin/tasks/flutter_tool_startup.dart @jensjoha @flutter/tool
```

Comment:
One entry should be enough.

Reply:
So I thought that but there are some more duplicates in the file. And I remember you saying it had to be one to one with the targets.

Reply:
The one to one map is between the entry here and the `task_name`, instead of the target (sorry if I said mapping to a target). See https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/request_handlers/flaky_handler_utils.dart#L221, which scan through the section of all devicelab entries.

Reply:
Ah okay. Thank you for the clarification. I will remove the duplicate.